### ISSUE TITLE
reverse sniper rifle

### ISSUE THREAD

**Issue by nosoytunose** [bluemagic123/tModLoader#2303]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

made some testing with the sniper rifle, and using the rifle scope nerfs the range of the zoom of the sniper itself, you get more range without it, to be sure, i actually deactivated all mods, to be sure, nothing changes,went to vanilla to make sure isn't a resolution problem,and it actuallly works as intened

### Steps to reproduce

1.equipe rifle scope
2.zoom with sniper rifle
3.thats it,just compare it

### Expected Behavior

extend zoom

### Actual Behavior

decrease zoom

**Comment by Setnour6** [comment]
Sorry for all the commit stuff here. I tried fixing it and then realized that the commit merged to one of the other commits I made, which was in a Pull Request, so I tried to undo it and then redo it again, but it pretty much broke the PR and the reasoning of it, so I closed the PR I made and made a new one, but with only the changes I once made before these changes.

As for this issue about the sniper and the scope, I experienced the same issue, and have tried fixing it, taking note of the commits. Here is what I did:
![image](https://user-images.githubusercontent.com/70764728/164993443-7b9c56e7-b842-4fe0-95f0-032510603f1e.png)
I didn't add it to the new PR, and I also need Mirsario to look at it. I presume he knows how the zooming mechanic works.

Again, I'm sorry for the commits that happened here.
